discourse_migration/discourse-categories.md

```diff
@@ -0,0 +1,75 @@
+# Proposed Discourse Categories
+
+* Announcements
+
+* Releases
+
+* Clang Compiler & Tools
+  - Using Clang
+  - Building Clang
+  - clangd
+
+* Subprojects
+  - lld
+  - Flang
+  - lldb
+  - Static Analyzer
+
+* Code Generation
+  - AArch64
+  - AMDGPU
+  - ARC
+  - ARM
+  - AVR
+  - BPF
+  - CSKY
+  - Hexagon
+  - Lanai
+  - M68k
```

Comment:
We have a lot of backends, and some like Lanai, M68k, SystemZ are niche and will have very little public traffic.  I'd recommend we start with 8 or 10 of the big and common ones, and then have an other category.  We can "promote" things out of other when there is a lot of traffic in the Other category.

Reply:
As a starting point, I'd go with:

"AArch64", "ARM", "Mips", "PowerPC", "RISCV", "WebAssembly", "X86", and "Other".

Rationale that those are the ones (I'm aware of) that have more than one organization as the primary driver and that are moderately popular.  This will be a new thing for the community, so we can always add more if people squawk loudly.